<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SeriesDialog</class>
 <widget class="QDialog" name="SeriesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1043</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="6">
    <widget class="QScrollArea" name="saTitles">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="wTitles">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1019</width>
        <height>434</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_12">
       <item row="0" column="0">
        <layout class="QGridLayout" name="grlTitles"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="pbCheckAll">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Выбрать все</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/images/icons8-checked-checkbox-80.png</normaloff>:/images/icons8-checked-checkbox-80.png</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QComboBox" name="cbSorting">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>По алфавиту</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>По выбранным</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>По количеству кадров</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="pbUncheckAll">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Снять все</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/images/icons8-unchecked-checkbox-80.png</normaloff>:/images/icons8-unchecked-checkbox-80.png</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="lblSorting">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Сортировка</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SeriesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SeriesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
